Library/Formula/xorg-util-macros.rb

```diff
@@ -0,0 +1,19 @@
+class XorgUtilMacros < Formula
+  desc "Miscellaneous utility macros required by the X.org codebase."
+  homepage "http://www.x.org"
+  url "http://xorg.freedesktop.org/archive/individual/util/util-macros-#{version}.tar.gz"
+  version "1.19.0"
+  sha256 "0d4df51b29023daf2f63aebf3ebc638ea88efedfd560ab5866741ab3f92acaa1"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "true"
```

Comment:
> There is seriously nothing to test.

Typically we test build tools by using them on a minimal example, like compiling a "Hello World" program. In this case, you could make a trivial `configure.ac` that uses them and see if it compiles to  a `configure`.